src/solvers/refinement/string_refinement.cpp

```diff
@@ -1603,10 +1604,10 @@ exprt string_refinementt::instantiate(
   return implies_exprt(bounds, instance);
 }
 
-/// instantiate a quantified formula representing `not_contains` by substituting
-/// the quantifiers and generating axioms
-/// \par parameters: a quantified formula representing `not_contains`, and a
-/// list to which to add the created lemmas to
+/// Instantiates a quantified formula representing `not_contains` by
+/// substituting the quantifiers and generating axioms.
+/// \param [in] axiom: the axiom to instantiate
+/// \param [out] lemmas: the lemmas produced through instantiation
 void string_refinementt::instantiate_not_contains(
```

Comment:
I find it a bit confusing to have to functions with almost the same name, you can maybe inline this one where it is used.